### PR TITLE
A few quick updates.

### PR DESCRIPTION
**Pull request by pete-naylor**
Typos, adding us-east-2, clarity around region requirements for MVI beta.

### CODE REVIEW

File: docs/vector-index/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ import TabItem from '@theme/TabItem';
 
 # Getting started with Momento Vector Index
 
-If you're looking to jump in and get started with Momento Vector Index with an SDK, you've come to the right place.
+If you're looking to jump in and get started with Momento Vector Index with an SDK, you've come to the right place. Please note that Momento Vector Index is currently in beta. At this time, the beta is supported in AWS us-west-2, and to test the service you'll need an API key which is also created for that region.
```
Comment (malandis) [inline docs/vector-index/index.md:13]: This is mentioned already right under this :)

File: docs/leaderboard/index.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ pagination_next: null
 
 ## What is Momento Leaderboard?
 
-Momento Leaderboard is a purpose-built leaderboard that supports tens of millions of items and rapid ingestion/querying/updates. While many databases approximate leaderboards via general purpose data structures like sorted set or range queries, Momento Leaderboard is a first-class, full-fledged service that you can quickly and easily integrate with your application. 
+Momento Leaderboardd is a purpose-built leaderboard service that supports tens of millions of items and rapid ingestion/querying/updates. While many databases approximate leaderboards via general purpose data structures like sorted set or range queries, Momento Leaderboards is a first-class, full-fledged service that you can quickly and easily integrate with your application.
```
Comment (allenheltondev) [inline docs/leaderboard/index.md]: Come on, Pete! "Leaderboardd"?

File: docs/leaderboard/index.md [hunk 3]
```diff
@@ -54,14 +54,14 @@ See the [API reference](./develop/api-reference/index.md) page for documentation
...
 Check out the [Node.js example](https://github.com/momentohq/client-sdk-javascript/blob/main/examples/nodejs/cache/leaderboard.ts) for a runnable program that includes examples of how to call each API.
 
-## Frequently asked questions about Momento Topics
+## Frequently asked questions about Momento Leaderboards
 
 <details>
-  <summary>What's the difference between Momento Leaderboard and Redis Sorted Sets?</summary>
-  Momento Leaderboard has a much more efficient memory footprint, allowing us to scale to tens of millions of records, and the APIs were built with gaming use cases specifically in mind.
+  <summary>What's the difference between Momento Leaderboards and Momento Cache Sorted Sets?</summary>
+  Momento Leaderboards have a much more efficient memory footprint, allowing us to scale to tens of millions of elements, and the APIs were built with gaming use cases specifically in mind.
```
Comment (allenheltondev) [inline docs/leaderboard/index.md:61]: Since the service is singular, this really should be "has" and not "have"

File: docs/leaderboard/index.md [hunk 4]
```diff
@@ -54,14 +54,14 @@ See the [API reference](./develop/api-reference/index.md) page for documentation
...
 </details>
 
 <details>
-  <summary>Do Momento Leaderboard items expire?</summary>
-  No, Momento Leaderboard items have no TTL.
+  <summary>Do Momento Leaderboards items expire?</summary>
+  No, Momento Leaderboards items have no TTL.
```
Comment (allenheltondev) [inline docs/leaderboard/index.md:66]: This needs to be 7 days

**Review by allenheltondev — CHANGES_REQUESTED** [review]

**Comment by anitarua** [comment]
[This PR](https://github.com/momentohq/public-dev-docs/pull/479) will cover the leaderboard fixes mentioned in the slack thread.

I think you can remove the leaderboard edits from this PR

File: docs/vector-index/index.md (hunk 1 above)
Comment (pete-naylor) [reply docs/vector-index/index.md:13]: It is not clear enough, which is why we're seeing customers tripping over it. In fact, it also has wording that suggests that you should just choose the region that's closest.

File: docs/leaderboard/index.md (hunk 2 above)
Comment (pete-naylor) [reply docs/leaderboard/index.md]: Dang. Was meant to be an "s" - I swear. Thanks!

File: docs/leaderboard/index.md (hunk 4 above)
Comment (pete-naylor) [reply docs/leaderboard/index.md:66]: Leaving this for @anitarua to sort out.

**Pull request closed without merging**